docs/usage/configuration-options.md

```diff
@@ -994,17 +994,37 @@ If this option is enabled, reviewers will need to create a new PR if additional
 
 ## forkProcessing
 
-By default, Renovate will skip over any repositories that are forked if Renovate is using `autodiscover` mode.
-This includes if the forked repository has a Renovate config file in the repo, because Renovate can't tell if that file was added by the original repository or not.
-If you wish to enable processing of a forked repository by Renovate when autodiscovering, you need to add `"forkProcessing": "enabled"` to your repository config or run the CLI command with `--fork-processing=enabled`.
+By default, Renovate skips any forked repositories when in `autodiscover` mode.
+It even skips a forked repository that has a Renovate configuration file, because Renovate doesn't know if that file was added by the forked repository.
 
-<!-- prettier-ignore -->
-!!! note
-    Only the `onboardingConfigFileName` (which defaults to `renovate.json`) is supported for `forkProcessing`. You cannot use other filenames because Renovate will use the platform API to check only for the default filename.
+### Process a fork in `autodiscover` mode`
+
+If you want Renovate to run on a forked repository when in `autodiscover` mode then:
+
+- Add `"forkProcessing": "enabled"` to your repository config,
+- Or run the CLI command with `--fork-processing=enabled`
+
+### Process a fork in other modes
+
+If you're running Renovate in some other mode, for example when giving a list of repositories to Renovate, but want to skip forked repositories: set `"forkProcessing": "disabled"` in your _global_ config.
```

Comment:
I'm out of ideas on how to rewrite this to be shorter or spread over two sentences. Can you suggest a better wording?